near-sdk/src/environment/env.rs

```diff
@@ -2768,8 +2799,8 @@ mod tests {
 
         let promise_index = super::promise_batch_create(&"alice.near".parse().unwrap());
         let code = vec![0u8; 100]; // Mock contract bytecode
-        let code_hash = vec![0u8; 32]; // Mock 32-byte hash
```

Comment:
why do you remove `vec!` here but not for `code` variable?

Reply:
Well, it was required to change the `code_hash` to be an array of exactly 32 bytes, but for `code` it is fine to be any `&[u8]`, so both would work.